Phase-5/SearchEngineTests/InvertedIndexTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using NSubstitute;
+using SearchEngine;
+using SearchEngine.Interfaces;
+using Xunit;
+
+namespace SearchEngineTests
+{
+    public class InvertedIndexTests
+    {
+        private IReader _readerMock;
+        private IWordProcessor _wordProcessorMock;
+        private IInvertedIndex _invertedIndex;
+
+        public InvertedIndexTests()
+        {
+            _readerMock = Substitute.For<IReader>();
+            _wordProcessorMock = Substitute.For<IWordProcessor>();
+            
+            _readerMock.Read().Returns(GetContents());
+            _wordProcessorMock.ProcessWord("Video.").Returns("video");
+            _wordProcessorMock.ProcessWord("Online,").Returns("online");
+            _wordProcessorMock.ProcessWord("Theme;").Returns("theme");
+            _wordProcessorMock.ProcessWord("video").Returns("video");
+
+            _invertedIndex = new InvertedIndex(_readerMock, _wordProcessorMock);
+        }
+
+        [Fact]
+        public void TestGetTokens()
+        {
+            Assert.Equal(GetCorrectTokens(), _invertedIndex.GetTokens());
+        }
+
+        [Fact]
+        public void TestSetUpDictionary()
+        {
+            var expected = GetCorrectDictionary();
+            var actual = _invertedIndex.GetDictionary();
+            Assert.Equal(3, actual.Count);
+            Assert.Equal(expected.Keys, actual.Keys);
+            foreach (var (key, value) in expected)
+            {
+                Assert.Equal(value, expected[key]);
+            }
```

Comment:
you just comparing expected with expected.
the counter that explained in InvertedIndex.cs will cause problem here